src/log.cpp

```diff
@@ -245,6 +253,28 @@ class Worker
 		}
 	}
 
+	bool open()
```

Comment:
This function should be the one actually called `reopen`, judging by what it does. And the other function can be renamed to something like `send_reopen_signal`